spotify_confidence/analysis/bayesian/bayesian_base.py

```diff
@@ -57,7 +57,6 @@ def randomization_warning_decorator(f):
 
     @wraps(f)
     def wrapper(*args, **kwargs):
-
```

Comment:
There's bunch of empty lines removed because of a newer version of formatting tool black being used